packages/react-router-cloudflare/sessions/workersKVStorage.ts

```diff
@@ -36,8 +35,7 @@ export function createWorkersKVSessionStorage<
     cookie,
     async createData(data, expires) {
       while (true) {
-        let randomBytes = new Uint8Array(8);
-        crypto.getRandomValues(randomBytes);
```

Comment:
I noticed the way this was written differed slightly from the other implementations, and thought this might even be a bug until I looked up that `crypto.getRandomValues` mutates the arg in place. I think it's much clearer making this consistent and avoiding the explicit mutation.

Reply:
Yeah, agree that's clearer 👍